runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java

```diff
@@ -29,7 +29,36 @@
 import java.util.Map;
 import lombok.AccessLevel;
 import lombok.Getter;
-import static net.runelite.api.ObjectID.*;
+import static net.runelite.api.ObjectID.ASH_PILE;
```

Comment:
Since these are static imports, a wildcard import is fine: https://github.com/runelite/runelite/wiki/Code-Conventions#imports